### PR TITLE
[ALLUXIO-1836] Worker creates tiered storage files

### PR DESCRIPTION
**Pull request by calvinjia**
https://alluxio.atlassian.net/browse/ALLUXIO-1836

### CODE REVIEW

File: core/server/src/main/java/alluxio/worker/block/BlockWorker.java [hunk 1]
```diff
@@ -629,4 +631,19 @@ public FileInfo getFileInfo(long fileId) throws IOException {
       throw new IOException(e);
     }
   }
+
+  /**
+   * Creates a file to represent a block denoted by the given block path. This file will be owned
+   * by the Alluxio worker but have 777 permissions so clients can read and write to the file
```
Comment (yupeng9) [inline core/server/src/main/java/alluxio/worker/block/BlockWorker.java]: hmm 777 implies the file can be renamed or deleted?

Comment (calvinjia) [inline core/server/src/main/java/alluxio/worker/block/BlockWorker.java]: Normally yes, but not with stickybit on the directory. I've updated the javadoc.

**Comment by apc999** [comment]
+1 LGTM

**Comment by yupeng9** [comment]
LGTM

**Pull request merged**